andela/project-kinsan
# #130341341 login with jwt

#### What does this PR do?

This PR adds a new route for authenticating users and returns a JWT that would be used for further communication with the server.
#### Description of Task to be completed?

The task was to create a new route for authenticating users and implement a new token based authentication system.
#### How should this be manually tested?

Pull from this feature branch, start your local mongodb using the command `sudo service mongod start`, then start the app using the command `nodemon server.js`. Use a Postman or a similar tool to send a post request to `localhost:3000/api/auth/login` a JSON response should be received.
#### Any background context you want to provide?
#### What are the relevant pivotal tracker stories?
#130341341 - Users should receive a JWT on successful login
#### Screenshots (if appropriate)

![screenshot from 2016-09-27 07 22 56](https://cloud.githubusercontent.com/assets/10448811/18862345/5ceaf79e-8483-11e6-8e55-bee4cbbf50a3.png)
#### Questions:
